pkg/test/dualstack/cloud_providers_dualstack_test.go

```diff
@@ -176,6 +177,34 @@ func TestCloudClusterIPFamily(t *testing.T) {
 			skipNodes:           true,
 			skipHostNetworkPods: true,
 		},
+		{
+			cloudName: "openstack",
+			osNames: []string{
+				// "centos",
+				// "flatcar",
+				// "rhel",
+				// "sles",
```

Comment:
Please remove these comments, you should add them back once they are implemented. Not sure if there is any value in having commented out values here.

Reply:
Agree, just listing them does not bring that much benefit. I would propose to add reason, e.g. `centos - not working because of ...`, then it makes sense as documentation

Reply:
Will remove because reason  are not always available.